lib/panel-league/util.js

```diff
@@ -92,6 +92,32 @@ module.exports.floodFill = ((state, callback) => {
   }
 });
 
+function seedFill(source, target, x, y) {
+  const row = source[y];
+  if (row && row[x] === false && target[y][x]) {
+    source[y][x] = true;
+    seedFill(source, target, x-1, y);
```

Comment:
Poorly formatted operators. Operators should be separated from operands with whitespace.